README.md

```diff
@@ -1,4 +1,6 @@
-# Coverity on Polaris for Azure DevOps.
+# [DEPRECATED] Coverity on Polaris for Azure DevOps.
+
+**NOTE:** This extension has been deprecated and will not work after February 14, 2025. It is recommended that you migrate to our new <a href="https://marketplace.visualstudio.com/items?itemName=blackduck.blackduck-coverity-on-polaris">Black Duck Coverity on Polaris</a>. Instructions can be found <a href="https://documentation.blackduck.com/bundle/ci-integrations/page/topics/t_ado-migrate.html">here</a>
```

Comment:
Update this to:

```**IMPORTANT NOTE: This plugin has been deprecated and will not be maintained. It will not work after Coverity on Polaris 2024.12.0 release. It is recommended that you migrate to our new Black Duck Coverity on Polaris Extension. Details can be found [here](https://documentation.blackduck.com/bundle/ci-integrations/page/topics/c_pol-ado-rel-notes.html).**
```